packages/core/src/plugins/use-schema.ts

```diff
@@ -7,21 +7,3 @@ export const useSchema = (schema: any): Plugin => {
     },
   };
 };
-
-export const useLazyLoadedSchema = (schemaLoader: (context: Maybe<DefaultContext>) => any): Plugin => {
-  return {
-    onEnveloped({ setSchema, context }) {
-      setSchema(schemaLoader(context));
```

Comment:
This is sync why do we remove this?

Reply:
I have no idea what this even does and the type defs just scream delete to me :D `context: Maybe<DefaultContext>) => any` :D

Reply:
It allows you to pass different schemas on each request.
```ts
useLazyLoadedSchema(context => {
   if (context.request.headers.get('x-my-token') === 'x-value') {
      return secretschema;
   } else {
      return publicschema;
   }
})
```

Reply:
oh `LazyLoaded` indicated for me that stuff should be a promise, maybe we should rename it? `useConditionalSchema`?

Reply:
`useSchemaByContext` :D to make it more obvious